sds_data_manager/constructs/ialirt_processing_construct.py

```diff
@@ -266,6 +266,8 @@ def add_autoscaling(self, processing_name):
             self,
             f"AsgCapacityProvider{processing_name}",
             auto_scaling_group=auto_scaling_group,
+            enable_managed_termination_protection=False,
+            enable_managed_scaling=False,
```

Comment:
So does this mean there is no more auto-scaling? I don't know if that's necessary or anything for i-ALiRT, but if we're switching to manual scaling only that will require some kind of close monitoring for the first few months to make sure we're hitting the 5 minute requirement.

Reply:
Totally agree. But this part of the script will make it so that a new EC2 instance will be launched by the ASG if one is deleted, and ECS will reschedule the task on the new instance.

        self.ecs_service = ecs.Ec2Service(
            self,
            f"IalirtService{processing_name}",
            cluster=self.ecs_cluster,
            task_definition=task_definition,
            security_groups=[self.ecs_security_group],
            **desired_count=1,**
            vpc_subnets=ec2.SubnetSelection(
                subnet_type=ec2.SubnetType.PRIVATE_WITH_EGRESS
            ),
        )
        
   Just to make certain I tested it. Attached are screenshots.

Reply:
That's good, but does I-ALiRT ever need more than one EC2 instance? With autoscaling off, it will never launch new ones even if the existing one(s) are maxed out on CPU or something. So, you would be limited to one EC2 even if more could be needed. That's my main concern.